test/unit/spec/signaling/v2/peerconnection.js

```diff
@@ -874,7 +898,18 @@ describe('PeerConnectionV2', () => {
             break;
         }
 
-        rev = test.pcv2._lastStableDescriptionRevision;
+        if (vmsFailOver && initial && type === 'offer' && signalingState === 'have-local-offer') {
+          // in case of vms fail-over, new PC get to 'have-local-offer' state
+          // by VMS sending create-offer message. Which ends up setting a
+          // test.pcv2._descriptionRevision. lets simulate that.
+          lastOfferRevision = 25; // even though its 'initial' state - last offer will not be 1.
```

Comment:
What if `lastOfferRevision = 0`?

Reply:
@ceaglest - @innerverse confirmed that RSP starts at revision 0, so we should be safe with assumptions here.